tests/test_resolvers/test_select.py

```diff
@@ -0,0 +1,47 @@
+from unittest.mock import Mock
+
+from sceptre.resolvers import Resolver
+from sceptre.resolvers.select import Select
+
+
+class MyListResolver(Resolver):
+    def resolve(self):
+        return ["first", "second", "third"]
+
+
+class ItemResolver(Resolver):
+    def resolve(self):
+        return self.argument
+
+
+class TestSelect:
```

Comment:
I need to implement better error handling around missing indexes/keys.

Reply:
Pydantic has changed my life:

https://docs.pydantic.dev/